### DISCUSSION TITLE
What happend to nootedblue?

### DISCUSSION THREAD

**Discussion by dsmirc** [ChefKissInc/NootedRed#256] [General]
Hello, first off, I want to express my gratitude for all of your mind-blowing efforts in attempting something unprecedented. About a year ago, you began working on Nootedblue -bringing intel iris xe to macos-, and it was regularly updated. However, it suddenly vanished from every GitHub repository owned by you guys, It was essentially wiped off the face of the internet, which is quite confusing. I'm here to ask about this disappearance. Moreover, I hope to learn whether you're currently engaged in similar endeavors. If so, I wish you the best of luck.

**Comment by VisualEhrmanntraut** [comment]
It is very complicated to do this and we did not even have any testers. The team is only me and my friend. We are currently focusing on other things, such as bringing support for more wifi cards, fixing issues with the existing projects, creating a better Hackintosh guide, etc.

**Reply by lentas1** [reply]
ok, but dont delete it. You could just archive repos.

**Reply by VisualEhrmanntraut** [reply]
it's not deleted, we just made it private in the event we decide to work on it later.

**Reply by VisualEhrmanntraut** [reply]
(there was no code for iris xe, I think Zorm was trying to get some old chip he had working, so I don't know why you're so upset)

**Reply by AlfCraft07** [reply]
"bringing support for more wifi cards"
So, are you planning to make a Realtek internal Wi-Fi kext?

**Reply by VisualEhrmanntraut** [reply]
Yes, me and my friend have been working on proper Wi-Fi drivers (instead of the hacks AirportItlwm does) and will have kexts for multiple chips (including Intel).

**Comment by AlfCraft07** [comment]
Does "Kexts for multiple chips" mean that, other than adding Realtek support and rewriting Intel drivers, you will also write drivers for Atheros chips like the QCA9377?

**Reply by VisualEhrmanntraut** [reply]
We plan to support Realtek, Intel, Ath10k (So those newer Qualcomm Atheros chips) and maybe Mediatek

**Reply by VisualEhrmanntraut** [reply]
And Realtek USB adapters will also work with this, as they use the same logic just through USB instead of PCI-e

**Reply by YoridanBG** [reply]
It appears that the chipset of the wifi card you have is supported by ath[10k](https://wireless.wiki.kernel.org/en/users/drivers/ath10k) and if you own this card I would suggest joining the cheffkiss telegram and contacting the developers directly so you can maybe be a beta tester in the future or giving a information that can help them.([link to the telegram](https://t.me/+Bx3MO9Hq8whhNzk9)
And hope you gave a good time!(I'm not affiliated with anyone in cheffkiss officially or unofficially)
~Yoridan

**Reply by AlfCraft07** [reply]
I don't own the card, but my friend does, anyway I'm still interested in this project because I have a Realtek RTL8821CE Wi-Fi adapter that doesn't work on macOS yet. I'm already in the telegram.

**Reply by YoridanBG** [reply]
Also i know this thread is not the right place to ask but also you(Visual) mentioned that MediaTek could be supported and if so will mt76 and mt7601u be supported?

**Reply by AlfCraft07** [reply]
It would also be useful for my two USB Wi-Fi cards: Mediatek MT7601U and Realtek RTL8192EU.

**Reply by YoridanBG** [reply]
The project will definitely be a much better solution than what airportitlw and i do too want to see how this project will end!

**Comment by SalmonPete** [comment]
So what's this?.

https://www.insanelymac.com/forum/topic/358305-80-solved-iris-xe-igpu-on-tiger-lake-successfully-loaded-icllp-frambuffer-and-vram-also-recognizes-1536mb-however-some-issues/

**Reply by VisualEhrmanntraut** [reply]
He cloned the source code before we made it private and he is experimenting with it. We do not have anything to do with any of that work. Unfortunately for them they do not realise the sheer amount of work required to get this working.

**Reply by VisualEhrmanntraut** [reply]
(If you are talking about some NootedBlue files floating around in there, obviously the WhateverGreen thing is just WhateverGreen with some minor changes)

**Comment by VisualEhrmanntraut** [comment]
Update: Zormeister left the team and I gave him some of the repositories out of my good will. One of them is NootedBlue. For future updates or any other information, if any, you will have to contact him.

EDIT: This does not mean ChefKiss will not release their own kexts similar to the 4 transferred to him.

**Comment by ghost** [comment]
I have a really stupid question. I wanna contribute to hackintosh community. But I have no idea where to start. I know python and can code with little help. I don't know how useful am I in this community. Help me.

**Comment by jalavoui** [comment]
this post is so funny
are you gonna reopen it?
@VisualEhrmanntraut

**Reply by VisualEhrmanntraut** [reply]
@jalavoui NootedBlue is property of Zormeister since his departure.

**Comment by oiutretu** [comment]
telegram expired
